### PR TITLE
Recruitment page remodelled 

### PR DESCRIPTION
**Pull request by Annnnnieee**
added jenny's recruitment page to the join us section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -414,72 +448,28 @@ <h1 class="team">Our Team</h2>
...
-
-        </div>
-        <a class="button" href="https://forms.gle/WatWQdgpyG4VzE9J6">APPLY</a>
+  <div class="join" id="join">
+    <div class="flex-container">
+      <div class="column" id="column-left">
+        <h2>UBC LAUNCH PAD RECRUITMENT 2019</h2>
+        <img
+          src="https://scontent.fyvr3-1.fna.fbcdn.net/v/t1.15752-9/69590466_938917156442770_5666590961975689216_n.png?_nc_cat=104&_nc_oc=AQkdBfHBYvbHqZJ0YbqUTRaAUW_Mk7-CgNzfq4j-w8PlQB39yhqHg5hF6pT_av2iaPo&_nc_ht=scontent.fyvr3-1.fna&oh=dee727ceee5214650cdb256c152386be&oe=5E123CAD">
+        <p>Are you a programmer, designer, or business student looking for an opportunity to work on interesting
```
Comment (Schemetrical) [inline index.html]: Can we add some spacing (possibly 8px or something) above this? Just looks a bit close right now

**Review by Schemetrical — APPROVED** [review]
Looks good, just a small nit. Also, the video is broken on my local but I assume it's working (might be an issue if you find it broken too). Otherwise lgtm

File: index.html [hunk 2]
```diff
@@ -31,7 +33,7 @@ <h1>
           <a id="nav-about" href="#about">ABOUT</a>
         </li>
         <li>
-          <a id="nav-projects" href="#projects">PROJECTS</a>
+          <a id="nav-projects" href="#projects">PROJ  ECTS</a>
```
Comment (Schemetrical) [inline index.html]: This might need to be undone

**Review by Schemetrical — CHANGES_REQUESTED** [review]
oops forgot about this. Just a mistype probably, easy fix

File: index.html (hunk 2 above)
Comment (Annnnnieee) [reply index.html]: oops! ha

**Comment by Annnnnieee** [comment]
looks like the reason the video wasn't playing was because mp4 isn't supported on a number of browsers. I changed it to webm. still wont work on opera, but no one uses opera

**Comment by Annnnnieee** [comment]
Also, jenny wanted the join section to stand alone instead of inline with the rest of the page, so I moved it out.

**Comment by Schemetrical** [comment]
I’ll check tomorrow sometime. This deploy preview thing is nice but I don’t see the video on mobile safari. Bout to take off :)

**Comment by Annnnnieee** [comment]
ahh.. looks like safari also doesn't support webm.

**Comment by Annnnnieee** [comment]
bye!

**Review by Schemetrical — APPROVED** [review]

**Pull request merged**